grammarinator/generate.py

```diff
@@ -270,6 +273,9 @@ def restricted_float(value):
     parser.add_argument('--version', action='version', version='%(prog)s {version}'.format(version=__version__))
     args = parser.parse_args()
 
+    if args.seed:
```

Comment:
To be on the safe side, this should be `if args.jobs == 1 and args.random_seed:`